.github/workflows/jit-security.yml

```diff
@@ -256,7 +256,7 @@ jobs:
       uses: jitsecurity-controls/jit-github-action@v4.2.2
       with:
         security_control: registry.jit.io/control-semgrep-alpine:latest
-        
+
```

Comment:
looks like there are lots of formatting changes, you can remove this from this PR